ocp-scripts/verify-jsl-expectations.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# This script verifies that the clone-project.sh and the export-/import
+# scripts used by it are not inadvertantly renamed.
+# ods-jenkins-shared-library depends on this.
+
+# The script is expected to be exucuted from the project root.
+
+set -e
+
+script_names=(clone-project.sh export-project.sh export-project.sh)
```

Comment:
Hi @henrjk , sorry for my delayed review!
question: isn't import-project.sh missing but export one duplicated in the list?